src/model/window-guardian.ts

```diff
@@ -66,18 +66,67 @@ const makeWindowGuardianConfig = (
     } as WindowGuardianConfig;
 };
 
+export const makeGuardianBrowserCAPI = (CAPI: CAPIType): CAPIBrowserType => ({
+    config: {
+        isDev: process.env.NODE_ENV !== 'production',
+        ajaxUrl: CAPI.config.ajaxUrl,
+        shortUrlId: CAPI.config.shortUrlId,
+        pageId: CAPI.config.pageId,
+        isPaidContent: !!CAPI.config.isPaidContent,
```

Comment:
Can we be explicit here using `===`?

Reply:
`CAPI.config.isPaidContent === true ? true : false` ?

Reply:
sorry yeah. Why is it not just  `isPaidContent: CAPI.config.isPaidContent`?

Reply:
I know mine and @oliverlloyd 's IDEs seem to not like using `.?` TS and flags it as an errors (prob a config issue on DCR). Also I assume if isPaidContent is undefined it should be the same as if it is false?